modules/stochastic_tools/test/tests/samplers/latin_hypercube/tests

```diff
@@ -75,6 +75,7 @@
     command = "./latin_hypercube.py"
     check_files = latin_hypercube.pdf
     required_python_packages = 'pandas numpy matplotlib'
+    valgrind = NONE
```

Comment:
what s the concern here

Reply:
running python doesn't work with valgrind